### PR TITLE
Merge 1.7.6.x into 1.7.7.x - 11/01/2020 

### PR DESCRIPTION
**Pull request by matks**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.7.x
| Description?  | Merge last changes from 1.7.7.x to develop
| Type?         | improvement
| Category?     | ME
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | 
| How to test?  |  No QA needed

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/17186)

### CODE REVIEW

File: install-dev/data/xml/tab.xml [hunk 1]
```diff
@@ -205,7 +205,7 @@
                     <class_name>AdminThemesCatalog</class_name>
                 </tab>
                 <tab id="Mail_Theme_Parent" id_parent="Look_feel" active="1" enabled="1" hide_host_mode="0" icon="">
-                  <class_name>AdminMailThemeParent</class_name>
+                  <class_name>AdminParentMailTheme</class_name>
```
Comment (matks) [inline install-dev/data/xml/tab.xml:208]: @atomiix Can you confirm this is expected ?

File: themes/classic/assets/js/theme.js [hunk 2]
```diff
@@ -1 +1,5 @@
...&&n.options.badge?0==n.$elementFilestyle.find(".badge").length?n.$elementFilestyle.find("label").append(' <span class="badge">'+t.length+"</span>"):0==t.length?n.$elementFilestyle.find(".badge").remove():n.$elementFilestyle.find(".badge").html(t.length):n.$elementFilestyle.find(".badge").remove()}),window.navigator.userAgent.search(/firefox/i)>-1&&n.$elementFilestyle.find("label").click(function(){return n.$element.click(),!1})}};var i=t.fn.filestyle;t.fn.filestyle=function(e,i){var r="",o=this.each(function(){if("file"===t(this).attr("type")){var o=t(this),a=o.data("filestyle"),s=t.extend({},t.fn.filestyle.defaults,e,"object"==typeof e&&e);a||(o.data("filestyle",a=new n(this,s)),a.constructor()),"string"==typeof e&&(r=a[e](i))}});return void 0!==typeof r?r:o},t.fn.filestyle.defaults={buttonText:"Choose file",iconName:"glyphicon glyphicon-folder-open",buttonName:"btn-default",size:"nr",input:!0,badge:!0,icon:!0,buttonBefore:!1,disabled:!1,placeholder:""},t.fn.filestyle.noConflict=function(){return t.fn.filestyle=i,this},t(function(){t(".filestyle").each(function(){var e=t(this),n={input:"false"!==e.attr("data-input"),icon:"false"!==e.attr("data-icon"),buttonBefore:"true"===e.attr("
```
Comment (matks) [inline themes/classic/assets/js/theme.js]: ⚠️ 

@NeOMakinG I tried to rebuild these assets but I failed to find the root folder 🤔 can you help me ?

**Comment by matks** [comment]
Cannot merge now, needs help on https://github.com/PrestaShop/PrestaShop/pull/17186/files?w=1#r366912485

EDIT: fixed

File: themes/classic/assets/js/theme.js (hunk 2 above)
Comment (PierreRambaud) [reply themes/classic/assets/js/theme.js]: `./tools/assets/build.sh` if you want to be sure.
Otherwise have a look at `themes/classic/_dev` and `themes`

Comment (matks) [reply themes/classic/assets/js/theme.js]: Fixed, thanks @PierreRambaud

File: install-dev/data/xml/tab.xml (hunk 1 above)
Comment (PierreRambaud) [reply install-dev/data/xml/tab.xml:208]: Yes this is the expected value: https://github.com/PrestaShop/PrestaShop/commit/1f38d27d33916dbcc24ecc76d76894c5c42c9f10#diff-d2d10cd0f2a1915857d4e90f2386ac7a

**Review by PierreRambaud — APPROVED** [review]

**Comment by matks** [comment]
Thank you for review

**Pull request merged**